.github/workflows/ci.yml

```diff
@@ -205,6 +205,8 @@ jobs:
           APPLE_PASSWORD: ${{ secrets.APPLE_PASSWORD }}
           APPLE_TEAM_ID: ${{ secrets.APPLE_TEAM_ID }}
         with:
+          includeDebug: ${{ github.event_name == 'release' || github.event_name == 'schedule' }}
```

Comment:
Does this actually evaluate as true/false in yaml?

Reply:
Yup should! And yes, morning matcha before code haha